### PR TITLE
Fix wrongly marked required arguments.

### PR DESCRIPTION
**Pull request by mariux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,12 +89,12 @@ See [variables.tf] and [examples/] for details and use-cases.
 
 ##### VPC
 
-- **`vpc_name`**: *(Required `string`)*
+- **`vpc_name`**: *(Optional `string`)*
 
   The Name of the VPC. This will be used to tag resources with Names by default.
```
Comment (thenaturalist) [inline README.md:94]:
```suggestion
  The name of the VPC. This will be used to tag resources with names by default.
```

Comment (mariux) [reply README.md:94]: please do not add blocking reviews for changed out of scope.. Just create a PR.

Comment (mariux) [reply README.md]: fixed

Comment (thenaturalist) [reply README.md]: got it

**Comment by mariux** [comment]
:rice_cracker:

**Comment by mariux** [comment]
:hourglass_flowing_sand:

**Pull request merged**